shared/Cargo.toml

```diff
@@ -37,10 +38,6 @@ tracing-subscriber = "0.2"
 url = "2.2"
 warp = "0.3"
 web3 = { version = "0.16", default-features = false }
-num-bigint = "0.3"
-mockall = "0.9"
-thiserror = "1.0"
```

Comment:
I see that these were all removed, but I don't see that they were taken out from any import statements. Does this mean they were just never used? Note that I know, I am at least using mockall in the shared crate.

Reply:
All of these actually appear in the list above. It comes from here: https://github.com/gnosis/gp-v2-services/commit/7289681feba06aa23ec7601a764edf0c6d023142
It could be that GitHub shows only the merge conflicts in the merge commit: inserting the new lines above was successful and creates no diff, while these lines are a conflict and are shown (everything from here to EOF was written my original merge base).